filer/models/virtualitems.py

```diff
@@ -47,8 +47,9 @@ def logical_path(self):
 
 
 class UnfiledImages(DummyFolder):
```

Comment:
We should change the class name to ``UnsortedUploads`` as well (and everywhere it is used in python code).